galaxy/api/views/views.py

```diff
@@ -51,6 +51,13 @@
 from galaxy.main import models
 from galaxy.common import version, sanitize_content_name
 
+# monitor
```

Comment:
This module is a leftover of galaxy before refactoring. New views should not be added here.
Can you please move your code to a separate module (e.g. `galaxy/api/views/monitor.py`)?